### ISSUE TITLE
Failed to extract pretrained model from the archive latest_ckpt.pth.tar

### ISSUE THREAD

**Issue by c9o** [avalonstrel/GatedConvolution_pytorch#11]
Did I miss something? Always failed to decompress the .tar file

        tar: Unrecognized archive format
        tar: Error exit delayed from previous errors.

**Comment by huangqianfirst** [comment]
I have download and decompress the .tar file successed. I can send it to you.

**Comment by c9o** [comment]
Hi @huangqianfirst thank you so much. Please send it to jingtao.ch@gmail.com

**Comment by c9o** [comment]
Hi @huangqianfirst can you help to share your latest_ckpt.pth.tar with me? Thanks.

**Comment by beiweizai-Txy** [comment]
Hi @huangqianfirst can you help to share your latest_ckpt.pth.tar with me,too? Thanks.Please send it to tianxuyang@tongji.edu.cn

**Comment by huangqianfirst** [comment]
@jtetc  @TJU-Tianxy  .Has been send. Sorry for late reply.

**Comment by Rayna-Zhang** [comment]
@huangqianfirst  could you also send it to me？ Thanks. zhangrn@pcl.ac.cn

**Comment by huangqianfirst** [comment]
@Rayna-Zhang .Has been send.

**Comment by SunTongtongtong** [comment]
@huangqianfirst  Could you please send it to me also?(Much appreciated :p)

**Comment by SunTongtongtong** [comment]
@huangqianfirst Could you please send it to me also?(Much appreciated :p)   sunshitong315@gmail.com

**Comment by dengyueyun666** [comment]
@huangqianfirst could you also send it to me？ Thanks.  dengyueyun666@sina.com

**Comment by huangqianfirst** [comment]
@dengyueyun666 . Has been sent.

**Comment by dengyueyun666** [comment]
@huangqianfirst The file was out of date. Please upload the file again and send it to me. Thanks. dengyueyun666@sina.com

**Comment by huangqianfirst** [comment]
@dengyueyun666 . New has been sent.

**Comment by SunTongtongtong** [comment]
@huangqianfirst  Sorry to bother you, I haven't received yet.  0.0Email:  sunshitong315@gmail.com

**Comment by dengyueyun666** [comment]
@huangqianfirst Hi! I guess you have been used the pre-trained model to test, right?
I wonder how to prepare a mask pkl file. I don't know how to turn my mask png file to pkl file.
Can you give an example code?
Thanks!

**Comment by huangqianfirst** [comment]
@SunTongtongtong . Has been sent.

**Comment by janzd** [comment]
You're not supposed to extract it. That's how pytorch models are saved. It's a tar file. Just load it as it is.

**Comment by sohelmsc** [comment]
@huangqianfirst Could you please send me the new pretrained decompressed file. I am currently working on it. That would be great. Email: mbhuiyan@ualberta.ca

**Comment by sohelmsc** [comment]
Or anyone who received the pretrained model from huangqianfirst. Please send it to mbhuiyan@ualberta.ca. That would be much appreciated.

**Comment by Janekaka** [comment]
@huangqianfirst Sorry to bother you,Could you please send it to me also? Much appreciated. Email: kakacnn@163.com

**Comment by linghushaoxia** [comment]
> 
> 
> I have download and decompress the .tar file successed. I can send it to you.

Thanks! Please send the file to linghushaoxialove@163.com
@huangqianfirst

**Comment by linghushaoxia** [comment]
the file is not a tar file
`torch.save(saved_model,'{}/epoch_{}_ckpt.pth.tar'.format(log_dir, i+1))
        torch.save(saved_model, '{}/latest_ckpt.pth.tar'.format(log_dir, i+1))`

**Comment by sohelmsc** [comment]
@linghushaoxia , Could you please send me the model file if you have it to mbhuiyan@ualberta.ca. Thanks in advance.

**Comment by huangqianfirst** [comment]
hi guys,I have to say ,this model have no good performance, there's no need to test this model anymore. I have implemented this work : https://github.com/huangqianfirst/Free_from-Image-Inpainting-with-Gated-Convolution. But I can't release my code now.

**Comment by cyuyundd** [comment]
Hi @huangqianfirst can you help to share your latest_ckpt.pth.tar with me,too? Thanks.Please send it to 1096294595@qq.com

**Comment by huangqianfirst** [comment]
@cyuyundd.Sorry,it has been long time ago, the file has been lost.
